step-functions/nino_check.asl.json

```diff
@@ -176,14 +177,30 @@
         "value.$": "$.SecretString"
       }
     },
+    "Fetch Current Time For Request Sent": {
```

Comment:
Let's refactor this a little bit. We can modify the "Call Matching API" to return the start and end times needed for the TxMA events. This removes the need for the additional 2 Lambda calls to get the time. 

As TxMA events are asynchronous we don't need to do them 1 by 1. Instead, the `EventBridge: PutEvents` actually accepts an array of events https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEvents.html so the TxMA events for nino-check can be done in 1 single state.

In return, this means we only added 1 state to the state machine, keeping the complexity of multiple states/lambdas out.